docker-compose.yml

```diff
@@ -0,0 +1,47 @@
+version: "3.9"
+
+x-common-variables: &common-variables
+  POSTGRES_USER: ${POSTGRES_USER:-postgres}
+  POSTGRES_PASSWORD: ${POSTGRES_PASSWORD:-postgres}
+  POSTGRES_DB: ${POSTGRES_DB:-template}
+
+services:
+  dbpg: 
+    container_name: postgres
```

Comment:
I think we could have the prefix `kingdom_` or `"{PROJET_NAME}"` for each name of the containers, as u used in the `web` service

Reply:
Agreed!